Dockerfile

```diff
@@ -1,8 +1,8 @@
-FROM jinaai/jina:2.0
+FROM jinaai/jina:2.0.7
```

Comment:
```suggestion
FROM jinaai/jina:2.0.7-py37-perf
```

Reply:
Addressed, please see [9989a80](https://github.com/jina-ai/executor-image-normalizer/pull/16/commits/9989a80cb07014affdd8f45ac7e5b586ac406498) for the change. Thanks.